ospfd/ospfd.h

```diff
@@ -249,6 +249,8 @@ struct ospf {
 
 	struct thread *t_write;
 #define OSPF_WRITE_INTERFACE_COUNT_DEFAULT    20
+	struct thread *default_routemap_timer;
```

Comment:
this event needs to be canceled when the OSPF area is destroyed.

Reply:
good catch. it was a miss from my side.

Reply:
Done.